upup/pkg/fi/fitasks/keypair.go

```diff
@@ -51,6 +51,9 @@ type Keypair struct {
 	Subject string `json:"subject"`
 	// Type the type of certificate i.e. CA, server, client etc
 	Type string `json:"type"`
+	// KeypairType stores the api version of kops.Keyset.  We are storing this info in order to determine if kops
+	// is accessing legacy secrets that do not use keyset.yaml.
+	KeypairType string `json:"keypairType"`
```

Comment:
I would call it APIVersion or Format.  And then I'd say "This lets us convert the stored keypair to our preferred version."  I don't think we should say "We are storing this info" because we're not storing it - tasks are ephemeral

Reply:
Sure ... renaming